### ISSUE TITLE
[CI] Failures in JavaJodaTimeDuellingTests.testSamePrinterOutput

### ISSUE THREAD

**Issue by cbuescher** [elastic/elasticsearch#63459]
**Build scan**:
https://gradle-enterprise.elastic.co/s/z7uccjvjhfies

**Repro line**:
./gradlew ':server:test' --tests "org.elasticsearch.common.joda.JavaJodaTimeDuellingTests.testSamePrinterOutput" \
  -Dtests.seed=7D44BFE43DEA01C1 \
  -Dtests.security.manager=true \
  -Dtests.locale=sv \
  -Dtests.timezone=Europe/Ljubljana \
  -Druntime.java=11

**Reproduces locally?**:
yes 

**Applicable branches**:
7.x, 7.10

**Failure history**:
Several failures, started around yesterday, Oct 7th

**Failure excerpt**:
java.lang.IllegalArgumentException: Invalid format: [weekyear]: Illegal pattern component: r
	at __randomizedtesting.SeedInfo.seed([7D44BFE43DEA01C1:3AC81B868AA4F2AF]:0)
	at org.elasticsearch.common.joda.Joda.forPattern(Joda.java:281)
	at org.elasticsearch.common.joda.JavaJodaTimeDuellingTests.assertSamePrinterOutput(JavaJodaTimeDuellingTests.java:859)
	at org.elasticsearch.common.joda.JavaJodaTimeDuellingTests.testSamePrinterOutput(JavaJodaTimeDuellingTests.java:779)

**Comment by cbuescher** [comment]
`JavaJodaTimeDuellingTests.testDuellingFormatsValidParsing` also seems to exhibit similar problems:

```
java.lang.IllegalArgumentException: Invalid format: [weekyear]: Illegal pattern component: r
	at __randomizedtesting.SeedInfo.seed([AFCA3C44F23D6F2E:95D843A75A6A9BBB]:0)
	at org.elasticsearch.common.joda.Joda.forPattern(Joda.java:281)
	at org.elasticsearch.common.joda.JavaJodaTimeDuellingTests.assertSameDate(JavaJodaTimeDuellingTests.java:893)
	at org.elasticsearch.common.joda.JavaJodaTimeDuellingTests.testDuellingFormatsValidParsing(JavaJodaTimeDuellingTests.java:535)
```
Build: https://gradle-enterprise.elastic.co/s/mubgz5njectmq

**Comment by cbuescher** [comment]
Muted on 7.x and 7.10 with be001182ab6ecd0efa8b16209b4f6f9aa3a3e5a0  and 564823b00f8e5a.

**Comment by cbuescher** [comment]
@pgomulka probably something up your alley?

**Comment by pgomulka** [comment]
closed by #63462 and #63463
